packages/visual-editor/src/components/puck/cards/PersonCard.tsx

```diff
@@ -0,0 +1,358 @@
+      handleResolveFieldsForCollections(data, params);
+
+    // Unnecessary field updates can lead to the fields losing focus
+    if (shouldReturnLastFields) {
+      return params.lastFields;
+    }
+
+    // Update each subfield based on isCollection
+    return {
+      ...params.lastFields,
+      card: {
+        label: "Card",
+        type: "object",
+        objectFields: {
+          headshot: YextCollectionSubfieldSelector<any, ImageWrapperProps>({
+            label: "Headshot",
+            isCollection: isCollection,
+            filter: {
+              directChildrenOf: directChildrenFilter,
+              types: ["type.image"],
+            },
+          }),
+          name: YextCollectionSubfieldSelector<any, string>({
+            label: "Name",
+            isCollection: isCollection,
+            filter: {
+              directChildrenOf: directChildrenFilter,
+              types: ["type.string"],
+            },
+          }),
+          title: YextCollectionSubfieldSelector<any, string>({
+            label: "Title",
+            isCollection: isCollection,
+            filter: {
+              directChildrenOf: directChildrenFilter,
+              types: ["type.string"],
+            },
+          }),
+          email: YextCollectionSubfieldSelector<any, string>({
+            label: "Email",
+            isCollection: isCollection,
+            filter: {
+              directChildrenOf: directChildrenFilter,
+              types: ["type.string"],
+              allowList: ["emails"],
+            },
+          }),
+          phone: YextCollectionSubfieldSelector<any, string>({
+            label: "Phone",
+            isCollection: isCollection,
+            filter: {
+              directChildrenOf: directChildrenFilter,
+              types: ["type.phone"],
+            },
+          }),
+          cta: YextCollectionSubfieldSelector<any, CTAProps>({
+            label: "CTA",
+            isCollection: isCollection,
+            filter: {
+              directChildrenOf: directChildrenFilter,
+              types: ["type.cta"],
+            },
+          }),
+        },
+      },
+    } as Fields<PersonCardProps>;
+  },
+  defaultProps: {
+    card: {
+      headshot: {
+        field: "",
+        constantValueEnabled: true,
+        constantValue: {
+          image: {
+            field: "",
+            constantValueEnabled: true,
+            constantValue: {
+              url: PLACEHOLDER_IMAGE_URL,
+              width: 360,
+              height: 360,
+            },
+          },
+          layout: "auto",
+        },
+      },
+      name: {
+        field: "",
+        constantValue: "First Last",
+        constantValueEnabled: true,
+      },
+      title: {
+        field: "",
+        constantValue: "Associate Agent",
+        constantValueEnabled: true,
+      },
+      phone: {
+        field: "",
+        constantValue: "(202) 770-6619",
```

Comment:
This could be a real number. Use (555) 555-5555 instead. Any number that starts with 555 is fake which is why it's used in movies/shows a lot.